tasks/update-infra-deployments.yaml

```diff
@@ -9,19 +9,28 @@ spec:
       description: Script for changing the infra-deployments
     - name: ORIGIN_REPO
     - name: REVISION
+    - name: shared-secret
+      default: infra-deployments-pr-creator
   workspaces:
     - name: deploy-key
```

Comment:
Also please add comment that the workspace is only as placeholder and can be removed after it will be removed from consumers.